packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -254,79 +254,94 @@ const helperBehavior = {
       return;
     }
 
-    const liquidationMargin = state.factoryPowers
-      .getGovernedParams()
-      .getLiquidationMargin();
-
-    // ask to be alerted when the price level falls enough that the vault
-    // with the highest debt to collateral ratio will no longer be valued at the
-    // liquidationMargin above its debt.
-    const triggerPoint = ceilMultiplyBy(
-      highestDebtRatio.numerator, // debt
-      liquidationMargin,
-    );
-
-    // INTERLOCK NOTE:
-    // The presence of an `outstandingQuote` is used to ensure there's only
-    // one thread of control waiting for a quote from the price authority.
-    // All later callers will just update the threshold for the `outstandingQuote`.
-    // The first invocation will skip the next block and continue to below where
-    // the `outstandingQuote` is assigned. So, if there's an outstanding quote,
-    // reset the level. If there's no current quote (because this is the first
-    // vault, or because a quote just resolved), then make a new request to the
-    // priceAuthority, and when it resolves, liquidate the first vault.
-    if (state.outstandingQuote) {
-      // Safe to call extraneously (lightweight and idempotent)
-      E(state.outstandingQuote).updateLevel(
-        highestDebtRatio.denominator, // collateral
-        triggerPoint,
-      );
-      trace(
-        'updating level for outstandingQuote',
-        triggerPoint,
-        highestDebtRatio.denominator,
+    // INTERLOCK: the first time through falls to the `else` case to process
+    // liquidations. Everything just updates the liquidation threshold.
+    if (state.liquidationInProgress) {
+      // ask to be alerted when the price level falls enough that the vault
+      // with the highest debt to collateral ratio will no longer be valued
+      // at the liquidationMargin above its debt.
+      const govParams = state.factoryPowers.getGovernedParams();
+      const liquidationMargin = govParams.getLiquidationMargin();
+      const triggerPoint = ceilMultiplyBy(
+        highestDebtRatio.numerator, // debt
+        liquidationMargin,
       );
-      return;
-    }
 
-    if (state.liquidationInProgress) {
-      return;
+      // Update the current in-progress quote if there is one. Otherwise,
+      // the new threshold will be picked up by the next quote request.
+      if (state.outstandingQuote) {
```

Comment:
why calculate `triggerPoint` if there's no `oustandingQuote`?

I'd find this "return early" flow a clearer to follow because there are two reasons _not_ to "reschedule price check":
- if there's no `liquidationInProgress` return promise to start liquidation
- if there's no `outstandingQuote` return 
- (else) actually reschedule the price check